src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -346,8 +351,11 @@ class DemuxFastqs
  var fastqIncludeSampleBarcodes: Boolean = false,
  @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
  var illuminaFileNames: Boolean = false,
- @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header.")
- val keepOnlyPassing: Boolean = false
+
+ @arg(doc="Keep only passing filter reads if true, otherwise keep all reads. Passing filter reads are determined from the comment in the FASTQ header")
+ val keepOnlyPassing: Boolean = false,
+ @arg(doc="Keep only reads identified as control if true, otherwise keep all reads. Control reads are determined from the comment in the FASTQ header")
+ val keepControlReads: Boolean = false
```

Comment:
What's the expected default here?  Since we didn't have this option previously do you want to have the default be to retain control reads so by default we output everything that's input?
Either way, I think boolean options are best when they default to false, because then the user can just use `--thing` instead of `--thing=false`.  So here I think I would suggest we have:
`val [omit|skip|discard]ControlReads: Boolean = false`